miner/worker.go

```diff
@@ -82,7 +83,7 @@ type environment struct {
 	blobs    int
 	size     uint64
 
-	rules            params.Rules
+	rules            ethparams.Rules
```

Comment:
again this should only cover changing constans in params package, we should not change types